src/mongoose_acc.erl

```diff
@@ -35,15 +36,21 @@
 -export([update_stanza/2]).
 % Access to namespaced fields
 -export([
+         set/3,
          set/4,
+         set_permanent/3,
```

Comment:
I don't see it used in other place than tests. Is it meant to be used in a future PR?

Reply:
exactly, i believe it will be used by the future PRs. plus the interface is quite general and covered by test cases.